<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>ImageCropperDemo</class>
  <widget class="QWidget" name="ImageCropperDemo">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>900</width>
        <height>600</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Image Cropper Demo</string>
    </property>
    <layout class="QHBoxLayout" name="mainLayout">
      <item>
        <widget class="ImageCropperLabel" name="imgCropperLabel">
          <property name="minimumSize">
            <size>
              <width>600</width>
              <height>500</height>
            </size>
          </property>
          <property name="frameStyle">
            <number>1</number>
          </property>
        </widget>
      </item>
      <item>
        <layout class="QVBoxLayout" name="vLayout">
          <item>
            <layout class="QFormLayout" name="formLayout1">
              <item row="0" column="0">
                <widget class="QLabel">
                  <property name="text">
                    <string>Preview:</string>
                  </property>
                </widget>
              </item>
              <item row="0" column="1">
                <widget class="QLabel" name="labelPreviewImage"/>
              </item>
              <item row="1" column="0">
                <widget class="QLabel">
                  <property name="text">
                    <string>OriginalImage:</string>
                  </property>
                </widget>
              </item>
              <item row="1" column="1">
                <layout class="QHBoxLayout" name="hOriginalImagePathLayout">
                  <item>
                    <widget class="QLineEdit" name="editOriginalImagePath"/>
                  </item>
                  <item>
                    <widget class="QPushButton" name="btnChooseOriginalImagePath"/>
                  </item>
                </layout>
              </item>
              <item row="2" column="0">
                <widget class="QLabel">
                  <property name="text">
                    <string>OutputShape:</string>
                  </property>
                </widget>
              </item>
              <item row="2" column="1">
                <widget class="QComboBox" name="comboOutputShape"/>
              </item>
              <item row="3" column="0">
                <widget class="QLabel">
                  <property name="text">
                    <string>CropperShape:</string>
                  </property>
                </widget>
              </item>
              <item row="3" column="1">
                <widget class="QComboBox" name="comboCropperShape"/>
              </item>
              <item row="4" column="0">
                <widget class="QLabel">
                  <property name="text">
                    <string>FixedSize:</string>
                  </property>
                </widget>
              </item>
              <item row="4" column="1">
                <layout class="QHBoxLayout" name="hCropperFixedSizeLayout">
                  <item>
                    <widget class="QLineEdit" name="editCropperFixedWidth"/>
                  </item>
                  <item>
                    <widget class="QLineEdit" name="editCropperFixedHeight"/>
                  </item>
                </layout>
              </item>
              <item row="5" column="0">
                <widget class="QLabel">
                  <property name="text">
                    <string>MinimumSize:</string>
                  </property>
                </widget>
              </item>
              <item row="5" column="1">
                <layout class="QHBoxLayout" name="hCropperMinSizeLayout">
                  <item>
                    <widget class="QLineEdit" name="editCropperMinWidth">
                      <property name="text">
                        <string>8</string>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QLineEdit" name="editCropperMinHeight">
                      <property name="text">
                        <string>8</string>
                      </property>
                    </widget>
                  </item>
                </layout>
              </item>
            </layout>
          </item>
          <item>
            <spacer name="verticalSpacer1">
              <property name="orientation">
                <enum>Qt::Vertical</enum>
              </property>
            </spacer>
          </item>
          <item>
            <layout class="QFormLayout" name="formLayout2">
              <item row="0" column="0">
                <widget class="QLabel">
                  <property name="text">
                    <string>EnableOpacity:</string>
                  </property>
                </widget>
              </item>
              <item row="0" column="1">
                <widget class="QCheckBox" name="checkEnableOpacity"/>
              </item>
              <item row="1" column="0">
                <widget class="QLabel">
                  <property name="text">
                    <string>Opacity:</string>
                  </property>
                </widget>
              </item>
              <item row="1" column="1">
                <widget class="QSlider" name="sliderOpacity">
                  <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
          <item>
            <spacer name="verticalSpacer2">
              <property name="orientation">
                <enum>Qt::Vertical</enum>
              </property>
            </spacer>
          </item>
          <item>
            <layout class="QFormLayout" name="formLayout3">
              <item row="0" column="0">
                <widget class="QLabel">
                  <property name="text">
                    <string>ShowDragSquare:</string>
                  </property>
                </widget>
              </item>
              <item row="0" column="1">
                <widget class="QCheckBox" name="checkShowDragSquare"/>
              </item>
              <item row="1" column="0">
                <widget class="QLabel">
                  <property name="text">
                    <string>DragSquareEdge:</string>
                  </property>
                </widget>
              </item>
              <item row="1" column="1">
                <widget class="QLineEdit" name="editDragSquareEdge">
                  <property name="text">
                    <string>8</string>
                  </property>
                </widget>
              </item>
              <item row="2" column="0">
                <widget class="QLabel">
                  <property name="text">
                    <string>DragSquareColor:</string>
                  </property>
                </widget>
              </item>
              <item row="2" column="1">
                <layout class="QHBoxLayout" name="hDragSquareColorLayout">
                  <item>
                    <widget class="QLabel" name="labelDragSquareColor"/>
                  </item>
                  <item>
                    <widget class="QPushButton" name="btnChooseDragSquareColor"/>
                  </item>
                </layout>
              </item>
            </layout>
          </item>
          <item>
            <spacer name="verticalSpacer3">
              <property name="orientation">
                <enum>Qt::Vertical</enum>
              </property>
            </spacer>
          </item>
          <item>
            <layout class="QFormLayout" name="formLayout4">
              <item row="0" column="0">
                <widget class="QLabel">
                  <property name="text">
                    <string>ShowRectBorder:</string>
                  </property>
                </widget>
              </item>
              <item row="0" column="1">
                <widget class="QCheckBox" name="checkShowRectBorder"/>
              </item>
              <item row="1" column="0">
                <widget class="QLabel">
                  <property name="text">
                    <string>RectBorderColor:</string>
                  </property>
                </widget>
              </item>
              <item row="1" column="1">
                <layout class="QHBoxLayout" name="hRectBorderColorLayout">
                  <item>
                    <widget class="QLabel" name="labelRectBorderColor"/>
                  </item>
                  <item>
                    <widget class="QPushButton" name="btnChooseRectBorderCorlor"/>
                  </item>
                </layout>
              </item>
            </layout>
          </item>
          <item>
            <spacer name="verticalSpacer4">
              <property name="orientation">
                <enum>Qt::Vertical</enum>
              </property>
            </spacer>
          </item>
          <item>
            <layout class="QHBoxLayout" name="btnLayout">
              <item>
                <spacer name="horizontalSpacer1">
                  <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                  </property>
                </spacer>
              </item>
              <item>
                <widget class="QPushButton" name="btnSavePreview">
                  <property name="text">
                    <string>Save</string>
                  </property>
                </widget>
              </item>
              <item>
                <spacer name="horizontalSpacer2">
                  <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                  </property>
                </spacer>
              </item>
              <item>
                <widget class="QPushButton" name="btnQuit">
                  <property name="text">
                    <string>Quit</string>
                  </property>
                </widget>
              </item>
              <item>
                <spacer name="horizontalSpacer3">
                  <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                  </property>
                </spacer>
              </item>
            </layout>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <customwidgets>
    <customwidget>
      <class>ImageCropperLabel</class>
      <extends>QLabel</extends>
      <header>imageCropperLabel.h</header>
    </customwidget>
  </customwidgets>
  <resources/>
  <connections/>
</ui>